README.md

```diff
@@ -38,9 +38,9 @@ Less bugs = better experience = more players for surf!
 * Download and install [Metamod 1.11](https://www.sourcemm.net/downloads.php/?branch=stable)
 * Download and install [SourceMod 1.10](https://www.sourcemod.net/downloads.php?branch=stable) (latest stable) or [SourceMod 1.11](https://www.sourcemod.net/downloads.php?branch=master&all=1) (required for some recommended plugins)
 * Download latest [release](https://github.com/surftimer/SurfTimer/releases/latest) and upload all the files to your csgo server directory
-* Set up A MySQL Database (MySQL 5.7, MySQL 8+, MariaDB supported)
+* Set up A MySQL Database (MySQL 5.7, MySQL 8+, MariaDB supported) and [initialise a schema for surftimer](https://github.com/surftimer/SurfTimer/blob/master/scripts/mysql-files/fresh_install.sql)
 * Add a MySQL database called surftimer to csgo/addons/sourcemod/configs/databases.cfg
-* Ensure [End-Touch-Fix](https://github.com/rumourA/End-Touch-Fix) is loaded, this is required to ensure times are always accurate
+* Ensure [End-Touch-Fix](https://github.com/rumourA/End-Touch-Fix) is loaded, this is required to ensure times are always accurate (note: this should be included and loaded by SurfTimer automatically without installing separately)
```

Comment:
EndTouchFix is not included in Surftimer, it's included inside the build package and will loaded, which is it, as standalone Plugin. Note should something like "included inside the build package" or so.

Reply:
updated it - let me know if this is better :slightly_smiling_face: